zeebe/topology/src/test/java/io/camunda/zeebe/topology/ClusterTopologyManagerTest.java

```diff
@@ -286,6 +286,35 @@ void shouldCallListenerWhenInconsistentLocalStateDetected() {
     assertThat(oldTopology.get()).isEqualTo(initialTopology);
   }
 
+  @Test
+  void shouldNotDetectInconsistencyWhenLocalMemberHasLeft() {
```

Comment:
:thought_balloon: Totally overthinking this: The generalization seems to be that `ClusterTopology#advance` is only allowed to do cleanup up, i.e. not change the semantic value of the topology, and `isConflictingTopology` must follow the same logic to treat semantically equal topologies in the same way.

If we continue to extend `ClusterTopology#advance`, it might become useful to add property tests for this to verify that `isConflictingTopology` is never true when making changes, except when forcing.

Reply:
Sounds good. But right now, as you said, it might be a bit overkill. ;)